documents/payment.md

```diff
@@ -0,0 +1,467 @@
+```
+
+**Parameters:**
+
+| Name  | Type      | Description                                      |
+|-------|-----------|--------------------------------------------------|
+| from  | timestamp | timestamp after which the payments were created  |
+| to    | timestamp | timestamp before which the payments were created |
+| count | integer   | number of payments to fetch (default: 10)        |
+| skip  | integer   | number of payments to be skipped (default: 0)    |
+
+**Response:**
+```json
+{
+  "entity": "collection",
+  "count": 2,
+  "items": [
+    {
+      "id": "pay_G8VaL2Z68LRtDs",
+      "entity": "payment",
+      "amount": 900,
+      "currency": "INR",
+      "status": "captured",
+      "order_id": "order_G8VXfKDWDEOHHd",
+      "invoice_id": null,
+      "international": false,
+      "method": "netbanking",
+      "amount_refunded": 0,
+      "refund_status": null,
+      "captured": true,
+      "description": "Purchase Shoes",
+      "card_id": null,
+      "bank": "KKBK",
+      "wallet": null,
+      "vpa": null,
+      "email": "gaurav.kumar@example.com",
+      "contact": "+919999999999",
+      "customer_id": "cust_DitrYCFtCIokBO",
+      "notes": [],
+      "fee": 22,
+      "tax": 4,
+      "error_code": null,
+      "error_description": null,
+      "error_source": null,
+      "error_step": null,
+      "error_reason": null,
+      "acquirer_data": {
+        "bank_transaction_id": "0125836177"
+      },
+      "created_at": 1606985740
+    }
+  ]
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Fetch a payment
+
+```rb
+paymentId = "pay_G8VQzjPLoAvm6D"
+
+Razorpay::Payment.fetch(paymentId)
+```
+
+**Parameters:**
+
+| Name       | Type   | Description                       |
+|------------|--------|-----------------------------------|
+| paymentId* | string | Id of the payment to be retrieved |
+
+**Response:**
+```json
+{
+  "id": "pay_G8VQzjPLoAvm6D",
+  "entity": "payment",
+  "amount": 1000,
+  "currency": "INR",
+  "status": "captured",
+  "order_id": "order_G8VPOayFxWEU28",
+  "invoice_id": null,
+  "international": false,
+  "method": "upi",
+  "amount_refunded": 0,
+  "refund_status": null,
+  "captured": true,
+  "description": "Purchase Shoes",
+  "card_id": null,
+  "bank": null,
+  "wallet": null,
+  "vpa": "gaurav.kumar@exampleupi",
+  "email": "gaurav.kumar@example.com",
+  "contact": "+919999999999",
+  "customer_id": "cust_DitrYCFtCIokBO",
+  "notes": [],
+  "fee": 24,
+  "tax": 4,
+  "error_code": null,
+  "error_description": null,
+  "error_source": null,
+  "error_step": null,
+  "error_reason": null,
+  "acquirer_data": {
+    "rrn": "033814379298"
+  },
+  "created_at": 1606985209
+}
+```
+-------------------------------------------------------------------------------------------------------
+
+### Fetch payments for an order
+
+```rb
+orderId = "order_DovFx48wjYEr2I"
+
+Razorpay::Payment.fetch(orderId).payments
```

Comment:
this is not implemented

Reply:
This one is part of order module ( fetch all payment of that order id)

https://github.com/razorpay/razorpay-ruby/blob/update_order/documents/order.md#fetch-payments-for-an-order

Reply:
This one is part of order module ( fetch all payment of that order id)

https://github.com/razorpay/razorpay-ruby/blob/update_order/documents/order.md#fetch-payments-for-an-order